### PR TITLE
Out of Range Options, PowerPredicton, and Combo Point Frame

### PR DESCRIPTION
**Pull request by Grimsbain**
Added option to show out of range coloring on the full icon or just on the hotkey.

### CODE REVIEW

File: Interface/AddOns/nMainbar/config.lua [hunk 1]
```diff
@@ -8,6 +8,7 @@ nMainbar.Config = {
         showVehicleKeybinds = true,
         showKeybinds = false,
         showMacronames = false,
+        OutOfRangeType = 'icon',     -- Changes how out of range coloring is displayed. Options: icon, hotkey.
```
Comment (renstrom) [inline Interface/AddOns/nMainbar/config.lua]: To align with the rest of the options:

```diff
-OutOfRangeType = 'icon',
+outOfRangeType = 'icon',
```

Comment (Grimsbain) [inline Interface/AddOns/nMainbar/config.lua]: Fixed

Comment (renstrom) [inline Interface/AddOns/nMainbar/config.lua]: Close, but no cigar :) You still have uppercase where you access it.

Comment (Grimsbain) [inline Interface/AddOns/nMainbar/config.lua]: It is corrected just not in this commit. If you show all the changes it is in lowercase.

File: Interface/AddOns/nMainbar/modules/skins.lua [hunk 2]
```diff
@@ -276,7 +276,12 @@ hooksecurefunc('ActionButton_OnUpdate', function(self, elapsed)
         if ( hotkey:GetText() == RANGE_INDICATOR ) then
             if ( valid == false ) then
                 hotkey:Show()
-                hotkey:SetVertexColor(unpack(cfg.color.OutOfRange))
+                if ( cfg.button.OutOfRangeType == 'icon' ) then
```
Comment (renstrom) [inline Interface/AddOns/nMainbar/modules/skins.lua]:
```diff
-if ( cfg.button.OutOfRangeType == 'icon' ) then
+if ( cfg.button.outOfRangeType == 'icon' ) then
```

**Pull request merged**